src/components/main/EditorPicks.tsx

```
@@ -113,43 +110,30 @@ export default function EditorPicks({ projects, isLoading }: {projects: MainProj
               transform: `translateX(-${currentPage * 100}%)`,
             }}
           >
-
-
-            {/* Pages Container */}
-            {pages.map((pageItems, pageIndex) => (
+            {pages.map((pageItems, pageIndex) => 
               <div key={pageIndex} className="w-full flex-shrink-0">
```

Biome (1.9.4)

[error] Avoid using the index of an array as key property in an element.

This is the source of the key value.

The order of the items may change, and this also affects performances and component state.
Check the React documentation. 

(lint/suspicious/noArrayIndexKey)